CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+- Add support for upstream auth with ENV variables ([#339](https://github.com/rubygems/gemstash/pull/339), [@CiTroNaK][])
```

Comment:
Minor thing: The Rake task `changelog` has a `.citrus` grammar which looks for _full_ username links, rather than "reference-style links".

```suggestion
- Add support for upstream auth with ENV variables ([#339](https://github.com/rubygems/gemstash/pull/339), [@CiTroNaK](https://github.com/CiTroNaK))
```